AirCasting/SDSync/ViewsFlow/SDSyncRootView/SDSyncRootView.swift

```diff
@@ -10,7 +10,7 @@ struct SDSyncRootView: View {
     
     var body: some View {
         VStack(spacing: 40) {
-            ProgressView(value: 0.1)
+            ProgressView(value: 0.142)
```

Comment:
I'm not sure how you ended up with those values, but what do you think about changing them to something like `1.0/6` (where 6 is the number of screens in this flow and 1 is the number of the particular screen)? It would be more clear for a person reading the code in my opinion.